server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -230,67 +178,20 @@ protected MManager() {
         logger.info("create system folder {} failed.", schemaFolder.getAbsolutePath());
```

Comment:
What will happen if this condition is triggered? 
It is a legacy issue but it is better to handle the scenario in this re-factor

Reply:
When trigger this condition, IOException will be thrown when init mmanager, since the mlog or tagFile won't be found.